conmon/conmon.c

```diff
@@ -171,6 +196,7 @@ static GOptionEntry opt_entries[] =
   { "log-size-max", 0, 0, G_OPTION_ARG_INT64, &opt_log_size_max, "Maximum size of log file", NULL },
   { "socket-dir-path", 0, 0, G_OPTION_ARG_STRING, &opt_socket_path, "Location of container attach sockets", NULL },
   { "version", 0, 0, G_OPTION_ARG_NONE, &opt_version, "Print the version and exit", NULL },
+  { "log-level", 0, 0, G_OPTION_ARG_INT, &opt_log_level, "Print debug logs based on log level", NULL },
```

Comment:
We should read this as ARG_STRING and map it to int for log_level.

Reply:
But then logrus already maps the levels to int, so isn't it better to use that instead of creating a new string to int map of the levels in conmon?

Reply:
Fixed. Reading as ARG_STRING and mapping to an int. Not the most high tech way, but it is the simplest solution I could think of. @mrunalp @runcom PTAL.